modules/apps/frontend-js/frontend-js-test/src/testFunctional/tests/TranslationManager.testcase

```diff
@@ -266,4 +266,64 @@ definition {
 		}
 	}
 
+	@description = "LPS-148144. Validates language status is not translated after re-add language on modal."
+	@priority = "3"
+	test HasNotTranslatedStatusAfterAddningLanguage {
```

Comment:
The main point of these two cases is whether saving or not after deleting a Translated language. It's better to reflect this in the names. Otherwise, they make me confused, should the status be Not Translated or Translated after adding a language and in which situation it's Translated in which situation it's Not Translated just by looking the names without knowing the steps.

And they check the status not after adding language, but re-adding it after deleting it. So I think the names are not appropriate.